docs/source/conf.py

```diff
@@ -0,0 +1,37 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+import sys
+
+sys.path.insert(0, "src")
+
+# -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+
+project = "rashdf"
+copyright = "2024, fema-ffrd"
+author = "fema-ffrd"
+release = "0.2.1"
+
+# -- General configuration ---------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#general-configuration
+
+extensions = [
+    "sphinx.ext.autodoc",
+    "sphinx.ext.autosummary",
+    "sphinx_rtd_theme",
+    "numpydoc",
+]
+
+templates_path = ["_templates"]
+exclude_patterns = []
+
+
+# -- Options for HTML output -------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#options-for-html-output
+
+# html_theme = "alabaster"
+html_theme = "sphinx_rtd_theme"
+html_static_path = ["_static"]
```

Comment:
I got this warning while building the docs:
`WARNING: html_static_path entry '_static' does not exist`

Per [this thread](https://stackoverflow.com/questions/70739677/sphinx-html-static-path-entry-does-not-exist), I created the `source/_static` directory and silenced it.